lib/models/MetricsModel.js

```diff
@@ -0,0 +1,37 @@
+'use strict'; // eslint-disable-line strict
+
+class MetricsModel {
+    /**
+     * Create a metrics object
+     * @param {number} ops - number of operations
+     * @param {number} bytes - data size in bytes
+     * @param {string} extension - extension
+     * @param {string} type - operation indicator (queued or processed)
+     * @param {string} bucket - bucket name
+     * @return {this} current instance
+     * @static
+     */
+
+    constructor(ops, bytes, extension, type, bucket) {
+        this._timestamp = Date.now();
+        this._ops = ops;
+        this._bytes = bytes;
+        this._extension = extension;
+        this._type = type;
+        this._bucket = bucket;
+        return this;
```

Comment:
Would be okay to not return anything in the constructor since the `new` keyword creates a new instance with `this` bound to the newly created object. So `new MetricsModel(...).serialize();` should still return the expected serialized object.